nautobot/utilities/testing/views.py

```diff
@@ -93,7 +94,7 @@ def model_to_dict(self, instance, fields, api=False):
             else:
 
                 # Convert ArrayFields to CSV strings
-                if type(instance._meta.get_field(key)) is ArrayField:
+                if type(instance._meta.get_field(key)) in (ArrayField, JSONArrayField):
```

Comment:
This would be a good time to change this to a dogmatic type check:
```suggestion
                if isinstance(instance._meta.get_field(key), (ArrayField, JSONArrayField)):
```